languages/elixir/exercises/concept/README.md

```diff
@@ -10,6 +10,7 @@ These are the concept exercises that have currently been implemented, as well as
 | -------------- | ---------------------------------------- | -------------------------------------------------------------- |
 | `basics`       | `basics`                                 | none                                                           |
 | `booleans`     | `booleans`                               | `basics`                                                       |
+| `numbers`      | `floating-point-numbers`                 | `basics`                                                       |
```

Comment:
If you agree with the above comment on the exercise also teaching integers, this can become:

```suggestion
| `numbers`      | `integers`<br/>`floating-point-numbers`  | `basics`                                                       |
```